Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -252,10 +253,10 @@ private async Task RegisterServerEndpoints()
             foreach (var address in addresses)
             {
                 // TODO: Remove this condition on platforms that support it
```

Comment:
huh, update the comment or remove?

Reply:
OK, I will append another commit to update the comment to keep same with `SsdpCommunicationsServer.CreateSocketAndListenForResponsesAsync`.